api/routes/family.py

```
@@ -190,11 +190,21 @@ async def import_families(
 
     family_layer = FamilyLayer(connection)
     reader = csv.reader(codecs.iterdecode(file.file, 'utf-8-sig'), delimiter=delimiter)
+    rows = [r for r in reader if not r[0].startswith('#')]
+
+    if len(rows) == 0:
+        if has_header:
+            return {
```

Rather than returning tuples here, I think it would be more correct to raise a HTTPException. There's some code here: https://github.com/populationgenomics/metamist/blob/dev/api/server.py#L149 that will take that exception and format the response in a consistent way.

---

Made the change.